src/quic/session.cc

```diff
@@ -363,7 +363,7 @@ Maybe<Session::Options> Session::Options::From(Environment* env,
 
   auto& state = BindingData::Get(env);
   auto params = value.As<Object>();
-  Options options;
+  Options options = Options();
```

Comment:
This seems odd. I might be missing something, but the explicit constructor call `Options()` should perform the same initialization as the implicit call in `Options options`, and assignment should not change anything either.

Reply:
@tniessen I'm not 100% sure this will fix the warning as I don't know that we can run coverity locally.  I think other similar reports have been resolved by a similar fix through. Do you have an alternate suggestion?

Reply:
@tniessen I'll commit to looking to make sure it resolved the report after this lands. If it does not I'll figure out some other change that should.  I'm thinking that is the easiest way to figure out if it does/does not unless you have another suggestion.

Reply:
@mhdawson Sorry, I missed the first ping. This still seems odd to me, especially given that coverity complains about `options.transport_params.preferred_address_ipv4._M_payload._M_payload` (that is, a field in a class that is not `Options`), but this PR has the necessary approvals to land and I won't block it.

Reply:
@tniessen you were correct. My change did not resolve the Coverity report. Will investigate further to better understand what is being reported.